compiler/noirc_frontend/src/elaborator/comptime.rs

```diff
@@ -151,9 +151,9 @@ impl<'context> Elaborator<'context> {
         };
 
         self.errors.extend(errors);
-        // Transfer the set of functions with errors from the fresh elaborator
-        // so the interpreter can skip calling them
-        self.functions_with_errors.extend(elaborator.functions_with_errors);
+        self.exprs_with_errors.extend(elaborator.exprs_with_errors);
+        self.stmts_with_errors.extend(elaborator.stmts_with_errors);
```

Comment:
If these were stored in the NodeInterner we wouldn't need to manually extend them here, no? Since we always want to just keep accumulating these and the NodeInterner stores ExprId-related things anyway, it seems like a good place for them.